dissect/target/plugins/os/windows/recyclebin.py

```diff
@@ -125,6 +129,44 @@ def read_bin_file(self, bin_path: TargetPath) -> RecycleBinRecord:
             _user=user,
         )
 
+    def read_recycle_deleted_folder(self, folder_path: TargetPath) -> RecycleBinRecord:
+        """
+        Generally speaking when deleting a file, the $R* file is the actual renamed deleted file.
+        This is however also the case for deleted folders. When a folder is deleted,
+        it is also renamed and placed here (with original recursive content).
+
+        This function will create RecycleBin records for each file in a deleted folder.
+        """
```

Comment:
```suggestion
        # Generally speaking when deleting a file, the $R* file is the actual renamed deleted file.
        # This is however also the case for deleted folders. When a folder is deleted,
        # it is also renamed and placed here (with original recursive content).
        #
        # This function will create RecycleBin records for each file in a deleted folder.
```
This reads more like a code comment than a docstring.